content/workshops/rhel_8/exercise1.1.adoc

```diff
@@ -0,0 +1,137 @@
+:package_url: http://docs.ansible.com/ansible/package_module.html
+:service_url: http://docs.ansible.com/ansible/service_module.html
+:yaml_url: http://docs.ansible.com/ansible/YAMLSyntax.html
+
+
+= Exercise 1.1 - Managing Updates Using the Web Console
+
+== Exercise Description
+
+Now that you've toured the web console, lets work through updating the system. 
+
+== Section 1: Basic Updating
+
+As you did in the previous section, navigate to the `Software Updates` section of the Web Console, by clicking on the `Software Updates` link, in the left-hand menu. There will then be a brief interval where `Refreshing package information` will appear, along with a progress bar. Once that completes, you can proceed with the exercise.
+
+=== Step 1: Check for available updates
+
+Take a look at the `Available Updates` section of the browser window.  You should see several packages listed, in a single group, like this:
+
+image::su_updates.png[Software Updates, 900, caption="Figure 1: ", title="Packages Available for Updating"]
+
+You can expand the list, to see more details, by clicking on the drill-down `>` link:
+
+image::su_updates_expanded.png[Software Updates, 900, caption="Figure 2: ", title="Expanded Updates"]
+
+In the expanded view, you can see details on what CVEs are relevant to the updates, and any associated bug IDs.  Additionally, a summary of the changes is displayed.
+
+=== Step 2: Install updates
+
+Once you are finished reading, click the `Install Security Updates` button, to install the updated packages.
+
+The updates will install, and while doing so, there will be information on what actions are taking place, and below that, an expandable log drill-down. Once complete, you may receive a window like this:
+
+image::su_updates_restart.png[Software Updates, 900, caption="Figure 3: ", title="Updates Recommend a Restart"]
+
+=== Step 2: Restart the system
+
+Click the `Restart Now` button, to reboot your managed host, and cause the changes to take effect. You will see screens like these, while the system is rebooting:
+
+image::su_updates_restarting.png[Software Updates, 900, caption="Figure 4: ", title="System Restarting"]
+
+image::su_updates_disconnected.png[Software Updates, 900, caption="Figure 5: ", title="Disconnected"]
+
+=== Step 3: Reconnect
+
+Wait for a couple of minutes, for the system to come back up, and then click the `Reconnect` button, to be returned to the login screen, for the web console. Login, as before.
+
+=== Step 4: Summary
+
+Once you are logged back in, you will again see the updates screen, with a list of the updates that you just applied. You can expand the history list, to see the details.
+
+image::su_updates_summary.png[Software Updates, 900, caption="Figure 6: ", title="Update Summary"]
+
+== Section 2: Automatic Updating
+
+Next, let's take a look at automatic updating.  There is an `Automatic Updates` switch, at the top of the updates window.
```

Comment:
Line 68 Next, we will discuss automatic updating. In the web console, you can choose to apply all updates, or security updates and also manage periodicity and time of your automatic updates.